packages/client/tests/e2e/_utils/standard.cmd.sh

```diff
@@ -19,6 +19,10 @@ export NODE_PATH=$(npm root --quiet -g) # make global packages available
 export NEXT_TELEMETRY_DISABLED=1
 export NO_COLOR=1
 
+# Ensure Bun is in PATH (in case it's not inherited from Dockerfile ENV)
```

Comment:
I believe the behavior of whether it's inherited or not should be well defined, so either this whole block is unnecessary, or the comment should be updated to be more assertive (e.g. "because it's not inherited...").